src/v1/device_manager_client.ts

```diff
@@ -951,23 +952,41 @@ export class DeviceManagerClient {
       {} | undefined
     ]
   > | void {
-    request = request || {};
-    let options: CallOptions;
-    if (typeof optionsOrCallback === 'function' && callback === undefined) {
-      callback = optionsOrCallback;
-      options = {};
-    } else {
-      options = optionsOrCallback as CallOptions;
-    }
-    options = options || {};
-    options.otherArgs = options.otherArgs || {};
-    options.otherArgs.headers = options.otherArgs.headers || {};
-    options.otherArgs.headers['x-goog-request-params'] =
-      this._gaxModule.routingHeader.fromParams({
-        name: request.name || '',
+
+    return new Promise(async (resolve, reject) => {
+      
+
+      var options = {
+        host: 'iot-sandbox.clearblade.com',
+        path: `/api/v/4/webhook/execute/`+ adminSystemKey +`/devices?name=`+request?.name,
```

Comment:
This should use a system key from a template system, not the admin system key. Also, the webhook is now called `cloudiot_devices`, not `devices`

Reply:
Updated